requirements.yml

```diff
@@ -3,6 +3,10 @@
 - src: openmicroscopy.postgresql
   version: 2.0.0
 
+- name: openmicroscopy.network
+  src: https://github.com/openmicroscopy/ansible-role-network/archive/1.0.0.tar.gz
```

Comment:
https://galaxy.ansible.com/openmicroscopy/network/ is now available.

Reply:
Updated